README.md

```diff
@@ -19,6 +19,15 @@ Resource your vimrc and execute `PlugInstall` to ensure you have the plugin
 installed.
 
 ### Setup
+
+### Repository
+
+This plugin assumes that you have a `bare` repository setup for your worktrees. Before creating them, be sure you to clone a `bare` version of your repository as shown below:
```

Comment:
This is true, but we are actively trying to change that.  Could you put in a quick tidbit about that?

Reply:
Alright - should be updated with a small "we will be changing this" part after.